src/scanner.c

```diff
@@ -2826,12 +2824,15 @@ Obj FuncALL_KEYWORDS(Obj self) {
   return l;
 }
 
-Obj FuncSET_PRINT_FORMATTING_STDOUT(Obj self, Obj val) {
-  if (val == False)
-      ((TypOutputFile *)(STATE(OutputFiles)+1))->format = 0;
-  else
-      ((TypOutputFile *)(STATE(OutputFiles)+1))->format = 1;
-  return val;
+Obj FuncSET_PRINT_FORMATTING_STDOUT(Obj self, Obj val)
+{
+    /* *stdout* is always OutputFiles[0] by convention */
+    if (val == False) {
+        STATE(OutputFiles)[0].format = 0;
+    } else {
+        STATE(OutputFiles)[0].format = 1;
+    }
```

Comment:
Perhaps even change this to `STATE(OutputFiles)[0].format = (val != False);`, or to avoid double negation: ``STATE(OutputFiles)[0].format = (val == True);`